pkgs/development/python-modules/torch/bin.nix

```diff
@@ -97,6 +97,23 @@ buildPythonPackage {
     addAutoPatchelfSearchPath "$out/${python.sitePackages}/torch/lib"
   '';
 
+  # See https://github.com/NixOS/nixpkgs/issues/296179
+  #
+  # This is a quick hack to add `libnvrtc` to the runpath so that torch can find
+  # it when it is needed at runtime.
+  extraRunpaths = [ "${lib.getLib cudaPackages.cuda_nvrtc}/lib" ];
```

Comment:
Aj, sorry I missed it in the previous round: this should be guarded behind `isLinux`, like the `buildInputs` are. Otherwise we'll try to evaluate the outPath for cuda_nvrtc on Darwin, which will fail

Reply:
Got it. Thanks!